apps/core/scenes/search/Search.js

```diff
@@ -2,17 +2,22 @@
 
 import * as React from 'react';
 import { View } from 'react-native';
+import { DateTime } from 'luxon';
```

Comment:
We will need to discuss if we want to go with `luxon` or `date-fns`. (see https://bundlephobia.com/result?p=luxon@1.9.0)

Reply:
You are right. `date-fns` is part of the default kiwi stack.
Have to admit that I simply used `luxon` because it was already in dependencies and is used in graphql part.

Reply:
@RobinCsl Do you want to have this `luxon` / `date-fns` dilemma resolved before we merge this request or can I make it part of next issue that I have mentioned in PR description (where I plan to expand `DatePicker` implementation little bit, which will mean shuffling with this date related parts little bit)?

Reply:
Next PR is fine for me, as `luxon` is already used somewhere else.
However, we should move away from `luxon` towards `date-fns` as soon as possible, to avoid other `luxon` usage.

Reply:
definitely, I also prefer `date-fns`